crates/embucket-lambda/Makefile

```diff
@@ -10,12 +10,17 @@ build:
 	cd ../.. && cargo lambda build --release --arm64 --manifest-path crates/embucket-lambda/Cargo.toml $(FEATURES_PARAM)
 
 # Deploy to AWS (must run from workspace root for include paths to work)
-deploy: build deploy-only
-	@echo "Deployed $(FUNCTION_NAME) to AWS"
+# Deploy to AWS (must run from workspace root for include paths to work)
+deploy: build
+    cd ../.. && cargo lambda deploy --binary-name bootstrap $(FUNCTION_NAME)
+    cd ../.. && cargo lambda deploy --layer-arn arn:aws:lambda:us-east-2:184161586896:layer:opentelemetry-collector-arm64-0_19_0:1 arn:aws:lambda:us-east-2:184161586896:layer:opentelemetry-collector-arm64-0_19_0:1 arn:aws:lambda:us-east-2:767397688925:layer:otel-collector-config:19
```

Comment:
This duplicate code is not necessary, pls check how it currently implemented in main branch